mp4parse_capi/src/lib.rs

```diff
@@ -581,6 +603,40 @@ pub unsafe extern fn mp4parse_is_fragmented(parser: *mut mp4parse_parser, track_
     MP4PARSE_OK
 }
 
+/// Get pssh system id and pssh box content for eme playback.
+#[no_mangle]
+pub unsafe extern fn mp4parse_get_pssh_info(parser: *mut mp4parse_parser, info: *mut mp4parse_pssh_info) -> mp4parse_error {
+    if parser.is_null() || info.is_null() || (*parser).poisoned() {
+        return MP4PARSE_ERROR_BADARG;
+    }
+
+    let context = (*parser).context_mut();
+    let pssh_data = (*parser).pssh_data_mut();
+    let info: &mut mp4parse_pssh_info = &mut *info;
+
+    pssh_data.clear();
+    for pssh in &context.psshs {
+        let mut data_len = Vec::new();
+        match data_len.write_u32::<byteorder::NativeEndian>(pssh.box_content.len() as u32) {
+            Err(_) => {
+                return MP4PARSE_ERROR_IO;
+            },
+            _ => (),
+        }
+        // pssh data format in gecko:
+        //   system_id
+        //   pssh size (in native endian)
+        //   pssh box content (including header)
```

Comment:
It might be better to put this description in the function-level doc-comment above, since it's something a caller would need to know, not just an internal implementation detail.